app/decorators/user_decorator.rb

```diff
@@ -35,8 +35,12 @@ def masked_two_factor_phone_number
     masked_number(user.phone)
   end
 
+  def identity_verified?
```

Comment:
Since `identity_not_verified?` is already public, seems very weird to me to only expose one? Going to leave as-is for now.